TypeCobol/Compiler/Directives/TypeCobolOptions.cs

```diff
@@ -33,8 +37,66 @@ public bool UseEuroInformationLegacyReplacingSyntax
             set { _useEuroInformationLegacyReplacingSyntax = value; }
         }
 
+        /// <summary>
+        /// Status code of these option.
+        /// </summary>
+        public ReturnCode OptionStatusCode
+        {
+            get;
+            private set;
+        }
```

Comment:
I'm not ok with this new property because it means that every client code instanciating a `TypeCobolOptions` should check the status and adapt. The simplest way to work around this is to consider the constructor always safe meaning here that when we fail to load the CPY file, we should just log and keep going with a null/empty CPY table (see my previous remark about the logic).

Reply:
property removed